diff -urN ipQalc/main.cpp ipQalc-patched/main.cpp
--- ipQalc/main.cpp	2010-04-22 20:36:34.000000000 +1100
+++ ipQalc-patched/main.cpp	2012-03-30 14:28:43.000000000 +1100
@@ -9,10 +9,9 @@
 
     QString locale = QLocale::system().name();
 
-    translator.load(QString("ipqalc_") + locale);
+    translator.load(QString("ipqalc_") + locale, "/usr/share/ipqalc");
     app.installTranslator(&translator);
 
-
     ipQalc form1;
     form1.show();
 
